2312/1/koans/about_scoring_project.rb

```diff
@@ -0,0 +1,109 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Greed is a dice game where you roll up to five dice to accumulate
+# points.  The following "score" function will be used to calculate the
+# score of a single roll of the dice.
+#
+# A greed roll is scored as follows:
+#
+# * A set of three ones is 1000 points
+#
+# * A set of three numbers (other than ones) is worth 100 times the
+#   number. (e.g. three fives is 500 points).
+#
+# * A one (that is not part of a set of three) is worth 100 points.
+#
+# * A five (that is not part of a set of three) is worth 50 points.
+#
+# * Everything else is worth 0 points.
+#
+#
+# Examples:
+#
+# score([1,1,1,5,1]) => 1150 points
+# score([2,3,4,6,2]) => 0 points
+# score([3,4,5,3,3]) => 350 points
+# score([1,5,1,2,4]) => 250 points
+#
+# More scoring examples are given in the tests below:
+#
+# Your goal is to write the score method.
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+# :reek:ControlParameter
+def cond_one(ins)
+  if ins == 1
+    1000
+  else
+    100 * i
+  end
+end
+
+# :reek:TooManyStatements
+# :reek:DuplicateMethodCall
+# :reek:FeatureEnvy
+def score(dice)
+  score = 0
+  totals = [0] * 7
+  dice.map { |ins| totals[ins] += 1 }
```

Comment:
each would do the same in this case but u actually need `inject` or `each_with_object` instead of two last calls